src/iuse_actor.h

```diff
@@ -813,6 +816,14 @@ class heal_actor : public iuse_actor
         float head_power = 0;
         /** How much hp to restore when healing torso? */
         float torso_power = 0;
+        /** How many intensity levels will be applied when healing limbs? */
+        float bandages_power = 0;
+        /** Extra intensity levels gained per skill level when healing limbs. */
+        float bandages_scaling = 0;
+        /** How many intensity levels will be applied when healing limbs? */
+        float disinfectant_power = 0;
+        /** Extra intensity levels gained per skill level when healing limbs. */
+        float disinfectant_scaling = 0;
```

Comment:
You could merge those into something like this:
```
struct scaled_effect_data {
    efftype_id effect;
    float base;
    float scaling;
};
std::vector<scaled_effect_data> scaled_effects;
```
Then, instead of checking `if( bandages_power > 0)`, you'd iterate over `scaled_effects` and apply all of those, without checks (checks could be done at load time).
This would be more generic, so if later someone wanted to add a new effect, say "immobilized", "moistened", "wound sewn" and so on, it wouldn't need any extra work to work.